test/models/checkList/technicalCompetence.check.test.js

```diff
@@ -49,8 +49,8 @@ lab.beforeEach(() => {
   sandbox = sinon.createSandbox()
 
   // Stub the asynchronous base methods
-  sandbox.stub(BaseCheck.prototype, 'getTechnicalCompetenceEvidence').value(() => fakeTechnicalCompetenceEvidence)
-  sandbox.stub(BaseCheck.prototype, 'getTechnicalManagers').value(() => fakeTechnicalManagersEvidence)
+  sandbox.stub(BaseCheck.prototype, 'getUploadedFileDetails').value(() => fakeTechnicalCompetenceEvidence)
+  sandbox.stub(BaseCheck.prototype, 'getUploadedFileDetails').value(() => fakeTechnicalManagersEvidence)
```

Comment:
This will need to be changed as the second stub just overrides the first one - the tests will only pass as both mock objects contain the exact same data.
Could use `onFirstCall` and `onSecondCall` or something like that to cater for the two individual calls.